setup/server/osx/mysql.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+brew install mysql
+pip install mysql-python --no-cache-dir
+brew services start mysql
```

Comment:
So you think it should be server/setup/osx/mysql.sh instead?

i wanted to separate the actual code from the utility scripts.